### ISSUE TITLE
Can't cast Pyroclasm when opponent doesn't have any creatures

### ISSUE THREAD

**Issue by itandetnik** [magarena/magarena#341]
I control some creatures that were temporarily taken over from computer by the ability of Tibalt, the Fiend-Blooded. The computer doesn't currently control any creatures.

I have Pyroclasm in hand, and enough lands to cast it. I want to play Pyroclasm in order to kill the creatures I currently control, before they return to the opponent. But the computer won't let me - the card is not highlighted and doesn't react to a click.

May be related to issue #266.

**Comment by ShawnieBoy** [comment]
It may be due to having 'Allow only sensible choices' checked in the gameplay -> general preferences.

That option puts the same targeting restriction on the Player as the AI uses. I'm surprised it would also use casting restrictions as well, but would make sense.

**Comment by itandetnik** [comment]
No, I have it off; always had.

**Comment by ShawnieBoy** [comment]
Thanks for the quick reply - I'll go investigate.

**Comment by ShawnieBoy** [comment]
Managed to duplicate it, I'm thinking the problem is more with Tibalt

**Comment by ShawnieBoy** [comment]
It does look like the AI casting restriction is being applied to the player - removing the AI cast restriction allows the spell to be cast while there are no creatures on the battlefield.

**Comment by PalladiaMors** [comment]
About "allow only sensible choices": it may be better to have that option off by default. I remember the first time I came across it I wanted to use Swords to Plowshares on one of my creatures and wasn't able to due to AI targeting restrictions. I had no idea that happened due to an option and thought it was a bug. The option's name is very counter-intuitive: it's very sensible to use swords to plowshares to gain life if you're about to die from damage. The human players know what they're doing when they target their own creatures with removal or damage or any other uncommon target choice, the restriction is really only required for the AI. The only advantage I see in having that option enabled is that it can speed up gameplay - but now Magarena has turn skipping, so that doesn't look as necessary anymore.

Tl; dr: in my opinion it's best to have allow only sensible choices off by default, and it might be a good idea to find a better name for the option because uncommon uses of cards can definitely be sensible (I suggest "Enable casting restrictions for human players")?

And sorry for hijacking the issue to discuss this since it's unrelated, just came to mind here...
